### PR TITLE
feat: take max batches count from contract

### PR DESCRIPTION
**Pull request by rkolpakov**

### CODE REVIEW

**Review by avsetsin — APPROVED** [review]
lgtm

File: src/services/withdrawal.py [hunk 1]
```diff
@@ -31,10 +31,10 @@ def get_finalization_batches(
         withdrawal_vault_balance: Wei,
         el_rewards_vault_balance: Wei
     ) -> list[int]:
-        is_paused = self._fetch_is_paused()
-        has_unfinalized_requests = self._has_unfinalized_requests()
+        if self._fetch_is_paused():
```
Comment (F4ever) [inline src/services/withdrawal.py:34]: Let's add the comment that we are not stopping txs here, we just don't eject

Comment (F4ever) [reply src/services/withdrawal.py:34]: I remember that we already discussed, but I don't remember this results of discussion

Comment (F4ever) [reply src/services/withdrawal.py:34]: Or we decide to stop tx?

**Comment by F4ever** [comment]
Check tests and there are some conflics

File: src/services/withdrawal.py (hunk 1 above)
Comment (rkolpakov) [reply src/services/withdrawal.py]: It's withdrawal finalization not ejecting. In this case we r not stopping tx just finalizing zero requests

**Pull request merged**